app/Region/Inverted/GanonsTower.php

```diff
@@ -23,7 +23,7 @@ public function initalize()
         $this->locations["Ganon's Tower - DMs Room - Top Left"]->setRequirements(function ($locations, $items) {
             return $items->has('Hammer')
                 && $items->has('Hookshot')
-                && ($items->has('MoonPearl')
+                && ($items->has('MoonPearl') || $this->world->config('canDungeonRevive')
```

Comment:
missed this earlier, but anytime we use `$this->world->config('XXX')` can we also have the default in there, e.g. ` $this->world->config('canDungeonRevive', false)`